### ISSUE TITLE
v1.3.0 Notes on SCM.ACTIVE_MEMBERSHIP

### ISSUE THREAD

**Issue by arc12** [jiscdev/analytics-udd#175]
I think these would benefit from some simplification/clarification. I think they arose from a skype between WIlbert and myself but without later reflection. I think we over-complicated things.

In particular, "which of these records is the last known to have been actively used." and "... be THE records with the most recent..." imply some THE-ness, singular courses.

I think the "last known" arose from a concern about the valididy period; that the ACTIVE indicator is referenced to a point in time, so that it gets over-written.

Bottom line is that this indicator is supposed to show clearly and simply whether, at the point of data extract, the student is considered to be "on the course". The value of this is that dates may not always be reliable. The student may be active on several concurrent courses.

probably delete some of the notes which try to explain possibilities.... they just confuse!

**Comment by alanepaull** [comment]
How about this revision:

###Purpose
To identify whether or not the student is studying or enrolled to study on the course, at the time the data is loaded into the LRW.

Then Valid Values table is:

1 - active; the student is studying on the course, or enrolled to study on the course.  
2 - not active; the student has finished the course, or has withdrawn from, left or transferred out of the course for any reason.
3 - unknown; it is not known whether the student is 'active' or 'not active' in relation to this course.

###Notes
A student may have sequential student_course_membership records with only 1 active record, or may be pursuing more than 1 course and therefore have more than 1 active record. When updating student_course_membership records, the ACTIVE_MEMBERSHIP property may need to be updated on more than one record.

**Comment by arc12** [comment]
looks good!

**Comment by alanepaull** [comment]
Updated as above.
